pkg/controllers/provisioning/scheduling/scheduler.go

```diff
@@ -137,6 +137,39 @@ func (r Results) NonPendingPodSchedulingErrors() string {
 	return msg.String()
 }
 
+func (r Results) NodeClaimsMeetingMinimumRequirements() Results {
```

Comment:
nit: Consider describing what this function does above it

Reply:
Done.